src/core/expression/qgsexpressionnodeimpl.h

```diff
@@ -398,6 +398,18 @@ class CORE_EXPORT QgsExpressionNodeCondition : public QgsExpressionNode
          */
         QgsExpressionNodeCondition::WhenThen *clone() const SIP_FACTORY;
 
+        /**
+         * The expression that makes the WHEN part of the condition.
+         * \return The expression node that makes the WHEN part of the condition check.
+         */
+        QgsExpressionNode *whenExp() const { return mWhenExp; }
```

Comment:
Should this return a const pointer? Or are we ok with the expression nodes being modified (without the expression itself being aware of this and invalidating any caches from previous preparations etc.)

Reply:
I think it should. Let me update it.  I think I just copied what was on the others but it was late.